### ISSUE TITLE
Net Force toggle options screen doesn't have size limit

### ISSUE THREAD

**Issue by BryceAG** [phetsims/forces-and-motion-basics#250]
related to phetsims/QA#57

On the Net force screen, the toggle options screen doesn't have a size limit. For string test long the mute button goes into the arrows for sum of forces:

![image](https://user-images.githubusercontent.com/15327201/32016954-7d3709a4-b982-11e7-9335-4d08933c12ba.png)

and it is even worse for string test xss, but it doesn't have to pass for that. For string test x, the mute and reset button are uncomfortably close:
![image](https://user-images.githubusercontent.com/15327201/32017054-c9c9047a-b982-11e7-883c-20767d4b6d25.png)

**Comment by jessegreenberg** [comment]
Fixed in the above commits, they need to be cherry-picked into 2.3 and 2.3-phetio.

**Comment by jessegreenberg** [comment]
Changes merged into the two release branches. Leaving open but fixed.

**Comment by phet-steele** [comment]
@jessegreenberg might as well improve panels on every screen? There's some overlap elsewhere (slider + applied force arrow):

<img width="961" alt="screen shot 2017-11-02 at 1 28 11 pm" src="https://user-images.githubusercontent.com/5863899/32346090-d32174da-bfd1-11e7-9b21-e2ce14aeaede.png">

**Comment by phet-steele** [comment]
To be clear, there are more extreme cases where overlap is going to happen anyway, but it shouldn't happen at the reasonable settings I have above.

This overlap is just going to happen...
<img width="946" alt="screen shot 2017-11-02 at 1 36 20 pm" src="https://user-images.githubusercontent.com/5863899/32346383-d62f0f4c-bfd2-11e7-8137-384a9d192792.png">

**Comment by phet-steele** [comment]
Actually @jessegreenberg, the image I have above (the extreme one) has the "Friction Force" string going outside the dev bounds. What do you think?

<img width="280" alt="screen shot 2017-11-02 at 1 40 09 pm" src="https://user-images.githubusercontent.com/5863899/32346570-5f7a4974-bfd3-11e7-8b9a-a127663f843a.png">

**Comment by jessegreenberg** [comment]
I reduced max width of text in the  motion  screen  panels to fix the case @phet-steele pointed out in https://github.com/phetsims/forces-and-motion-basics/issues/250#issuecomment-341532897.

@arouinfar and I talked about the case in https://github.com/phetsims/forces-and-motion-basics/issues/250#issuecomment-341535523.  In that case we don't care that the label is going off screen because you would see the label as the arrow grows. And since it would be harder to fix than the other labels we are going to let it go.

https://github.com/phetsims/forces-and-motion-basics/commit/be4cbedb60e803fad5b4b786738fb98c61ca63f7 needs to be cherry-picked into the release branches.
